services/s3/src/main/resources/codegen-resources/customization.config

```diff
@@ -172,5 +172,14 @@
       "clientConfiguration"
     ]
   },
+  "additionalBuilderMethods": [
+      {
+        "methodName": "crtClientBuilder",
+        "delegateBuilder": "builder",
```

Comment:
Do we need `delegateBuilder`? Can we assume the name is `builder`? We can add this customization if needed in the future, but I doubt we will ever need it.

Reply:
sure.. removed delegateBuilder